src/simulator_core/entities/heat_network.py

```diff
@@ -50,4 +62,18 @@ def store_output(self) -> None:
         not converged for the input of the controller.
         :return: None
         """
-        pass
+        for py_asset in self.assets:
+            py_asset.write_to_output()
+        for py_junction in self.junctions:
+            py_junction.write_to_output()
+
+    def gather_output(self) -> pd.DataFrame:
+        """Method to gather output of all assets and return it as a dict.
+
+        :return: Dict with for all asset a dict with all outputs.
+        """
+        # Todo what do we do with the junction output do we need it?
+        result = pd.DataFrame()
```

Comment:
Zou je niet alleen DataFrame en concat kunnen importeren vanuit pandas?

Reply:
Nu is duidelijker dat je bezig bent met een onderdeel uit de pandas module. als je alleen dataframe en concat importeert verlies je dat. Moeten wel even afspraak over maken om dit consistent te doen.

Reply:
Besproken we houden importeren hele module zie: https://stackoverflow.com/questions/60268690/pandas-importing-dataframe-vs-whole-library-speed-and-memory-concern

Reply:
#58